tracee-ebpf/Makefile

```diff
@@ -47,9 +44,9 @@ build: $(OUT_BIN)
 
 go_env := GOOS=linux GOARCH=$(ARCH:x86_64=amd64) CC=$(CMD_CLANG) CGO_CFLAGS="-I $(abspath $(LIBBPF_HEADERS))" CGO_LDFLAGS="$(abspath $(LIBBPF_OBJ))"
 ifndef DOCKER
-$(OUT_BIN): $(LIBBPF_HEADERS) $(LIBBPF_OBJ) $(filter-out *_test.go,$(GO_SRC)) $(BPF_BUNDLE) | $(OUT_DIR)
-	$(go_env) go build -v -o $(OUT_BIN) \
-	-ldflags "-X main.version=$(VERSION) -X main.bpf_core=$(CORE)"
+$(OUT_BIN): $(LIBBPF_HEADERS) $(LIBBPF_OBJ) $(OUT_BPF_CORE) $(OUT_BPF) $(filter-out *_test.go,$(GO_SRC)) $(BPF_BUNDLE) | $(OUT_DIR)
```

Comment:
I think that we should only add here OUT_BPF_CORE, shouldn't we?

Reply:
We can build the CO:RE object regardless of if BTF is enabled on the kernel, but that doesn't mean it can run it. So building the kernel specific one as a backup I think makes sense. WDYT?

Reply:
The all target (see the header of this makefile) already builds the bpf target, which compiles both of these.
So actually, you can remove them both from here

Reply:
Or not. Keep the core one, as it is necessary by the OUT_BIN.
Just remove the OUT_BPF as it is already built by the all and bpf targets